### PR TITLE
Merge login and signin

### LINKED ISSUE #8328: Merge Login & Signup step
Both login & signup user journeys are currently the same.

We can provide a single route for login and redirect the user to the signup page if the user is not yet registered

### PR DESCRIPTION
**Pull request by Abhi1992002**
### Fixes #8328 

Both the login and signup processes were the same, causing confusion. Now, there’s a single route for login, and if the user isn’t registered, they’ll be redirected to the signup page automatically.

### Changes 🏗️

- Created two separate files for login and signup.

- If a user tries to log in and their email is not found in the system, they will 
   be redirected to the signup page.
         
- If the user is already registered, the usual login process will proceed.

### Testing 🔍

• The login flow to ensure redirection to signup if email is not present.
• Confirmed successful login for registered users.

https://github.com/user-attachments/assets/0a398b4e-d57a-4634-be6c-d4a326fc9792

### CODE REVIEW

File: autogpt_platform/docker-compose.yml
Comment (ntindle) [inline autogpt_platform/docker-compose.yml:1]: revert these changes, not needed afaik (as far as I know)

File: autogpt_platform/supabase
Comment (ntindle) [inline autogpt_platform/supabase:1]: revert this plz

File: autogpt_platform/frontend/yarn.lock
Comment (ntindle) [inline autogpt_platform/frontend/yarn.lock:1]: I don't think this one is needed

**Review by ntindle — CHANGES_REQUESTED** [review]
good start, just some quick things. I'll let other team members review the frontend code otherwise

File: autogpt_platform/docker-compose.yml
Comment (Abhi1992002) [reply autogpt_platform/docker-compose.yml:1]: Done..!!

**Comment by Abhi1992002** [comment]
Done..!!

**Comment by aarushik93** [comment]
@Abhi1992002 same thing here regarding supabase

**Review by ntindle — APPROVED** [review]

**Pull request merged**